tox.ini

```diff
@@ -68,12 +43,13 @@ deps =
     flake8-docstrings
     flake8-copyright
     flake8-builtins
+    pyproject-flake8
     pep8-naming
     isort
 commands =
-    flake8 {toxinidir}/src {toxinidir}/tests {toxinidir}/lib/charms/alertmanager_k8s
-    isort --check-only --diff {toxinidir}/src/. {toxinidir}/tests/. {toxinidir}/lib/charms/alertmanager_k8s/.
-    black --check --diff --line-length=99 --target-version=py38 {toxinidir}/src {toxinidir}/tests {toxinidir}/lib/charms/alertmanager_k8s
+    pflake8 {[vars]all_path}
```

Comment:
Add comment about (temporary) use of pflake?

Reply:
Will do